bin/katello-certs-check

```diff
@@ -11,12 +11,15 @@ function usage () {
 Verifies, that custom SSL certificate files are usable
 as part of the Katello installation. When passing filenames use absolute paths.
 
-usage: $0 -c CERT_FILE -k KEY_FILE -b CA_BUNDLE_FILE
+usage: $0 -t [foreman|foreman-proxy] -c CERT_FILE -k KEY_FILE -b CA_BUNDLE_FILE
 HELP
 }
 
-while getopts "c:k:b:" opt; do
+while getopts "t:c:k:b:" opt; do
     case $opt in
+        t)
+            TARGET=`echo $OPTARG|tr '[:upper:]' '[:lower:]'`
```

Comment:
More a fan of `$()` since you can nest those and it's easy to see where it begins and ends.
```suggestion
            TARGET=$(echo $OPTARG|tr '[:upper:]' '[:lower:]')
```